code/modules/reagents/chemistry/reagents/impure_reagents.dm

```diff
@@ -23,6 +23,7 @@
 	name = "Toxic sludge"
 	description = "Toxic chemical isomers made from impure reactions. Causes toxin damage"
 	ph = 2
+	chemical_flags = REAGENT_CANSYNTH
```

Comment:
Will this override the parent's flags?
i.e. all children of impurity should have `chemical_flags = REAGENT_INVISIBLE | REAGENT_SNEAKYNAME | REAGENT_DONOTSPLIT` as well

Reply:
my brain smal

Reply:
Your brain strong!